### PR TITLE
added locust load test

### PR DESCRIPTION
**Pull request by ashagraw91**

### CODE REVIEW

**Comment by ypkang** [comment]
How do I run this? Can you add a quick README?

File: support/locust/install.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import sys
```
Comment (ypkang) [inline support/locust/install.py]: What's this for? Do we need this file?

File: support/locust/app.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+        # global UserName
+        users = list(reader)
+        return users
+
+
+UserName = add_users()
+
+
+def read_excel(columName):
+    df = pd.read_excel('qna.xlsx', sheet_name='Sheet1')  # can also index sheet by name or fetch all sheets
```
Comment (ypkang) [inline support/locust/app.py]: Let's use csv instead of spreadsheet.

**Review by ypkang — CHANGES_REQUESTED** [review]
Left some comments. @ashagraw91 please take a look.

File: support/locust/install.py (hunk 1 above)
Comment (ashagraw91) [reply support/locust/install.py]: removed, was added for small fix but resolved later

File: support/locust/app.py (hunk 2 above)
Comment (ashagraw91) [reply support/locust/app.py]: changed

**Comment by ypkang** [comment]
@ashagraw91 Hey Ashish, @Gorgeous-Patrick (Baichuan) has some bandwidth now to work on this so he is going to take this over and push it across the finish line.

**Comment by marsninja** [comment]
@ypkang We should close the loop on this commit, seems quite important to finalize. Lets have a chat about it in a special Jaseci paper meeting.

**Comment by ypkang** [comment]
superseded by #257. Closing.

**Pull request closed without merging**